publication-doi-commons/src/main/java/no/unit/nva/publication/doi/PublicationMapper.java

```diff
@@ -49,7 +55,7 @@ private static URI transformIdentifierToId(String namespace, String identifier)
      * Map a DynamodbStreamRecord with oldImage and/or newImage to PublicationMapping. Publication is a wrapper object
      * containing mapped old and/or new Publication.
      *
-     * @param streamRecord  DynamodbStreamRecord
+     * @param streamRecord DynamodbStreamRecord
      * @return PublicationMapping
      */
     public PublicationMapping fromDynamodbStreamRecord(DynamodbStreamRecord streamRecord) {
```

Comment:
note to self:

This method doesnt belong in this class I believe @janbj , this one should use our DAOs.

Only DAOs should access the DynamodbStreamRecord .

Reply:
We treat `DynamodbEvent` and `DynamodbEvent.StreamRecord` as our own data classes even if it comes from AWS library.